app/javascript/angular/code/services/google/custom_marker.js

```diff
@@ -55,5 +55,16 @@ export function buildCustomMarker(latLng, content, colorClass, callback, type) {
           });
         });
   };
-  return new CustomMarker(latLng, content, colorClass, callback, type);
+
+  CustomMarker.prototype.getPosition = function() {
+    return new google.maps.LatLng({ lat: this.position.lat(), lng: this.position.lng() });
+  };
+
+  CustomMarker.prototype.getDraggable = () => {};
```

Comment:
`getPosition` and `getDraggable` are required to make the clusterer library work.

Reply:
If you feel like it would help ourselves in the future maybe you could add a comment in the code to say that